src/engraving/libmscore/measure.cpp

```diff
@@ -4386,23 +4386,41 @@ void Measure::computeWidth(Fraction minTicks, double stretchCoeff)
     computeWidth(s, x, isSystemHeader, minTicks, stretchCoeff);
 
     // Check against minimum width and increase if needed
+    double minWidth = computeMinMeasureWidth();
+    if (width() < minWidth) {
+        setWidthToTargetValue(s, x, isSystemHeader, minTicks, stretchCoeff, minWidth);
+        setWidthLocked(true);
+    } else {
+        setWidthLocked(false);
+    }
+}
+
+double Measure::computeMinMeasureWidth() const
+{
     double minWidth = isMMRest() ? score()->styleMM(Sid::minMMRestWidth) : score()->styleMM(Sid::minMeasureWidth);
     double maxWidth = system()->width() - system()->leftMargin(); // maximum available system width (left margin accounts for possible indentation)
-
-    // System width may not yet be available for the linear mode (e.g. continuous view)
-    // Will use the minimum width from the style in this case
     if (maxWidth <= 0) {
+        // System width may not yet be available for the linear mode (e.g. continuous view)
+        // Will use the minimum width from the style in this case
         maxWidth = minWidth;
     }
-
     minWidth = std::min(minWidth, maxWidth); // Accounts for a case where the user may set the minMeasureWidth to a value larger than the available system width
```

Comment:
I tested it, and it seems this line is not entirely doing its job. With a high enough minimum measure width, measures can now become wider than the system, but only to a certain extent, not infinitely. 
<img width="400" alt="Schermafbeelding 2022-07-30 om 04 08 34" src="https://user-images.githubusercontent.com/48658420/181866244-433adce3-c9cf-4575-9125-7db9b0cabd6c.png">

Probably, the cause is that you return `minWidth + startPosition` at the bottom of the function, which is equal to `maxWidth + startPosition` in the case that `maxWidth` initially was smaller than `minWidth`.

I think the solution is to return `std::min(minWidth + startPosition, maxWidth)` instead, and drop the line `minWidth = std::min(minWidth, maxWidth);`.

Reply:
Good catch! Should be fixed now